e2e/policy/policy_test.go

```diff
@@ -175,13 +175,25 @@ func getFailures(ctx context.Context, t *testing.T, ct *contrasttest.ContrastTes
 	require.NoError(err)
 	failures := -1
 	for k, v := range metrics {
-		if k == "contrast_meshapi_attestation_failures_total" {
-			failures = int(v.GetMetric()[0].GetCounter().GetValue())
+		if k != "contrast_grpc_server_handled_total" {
+			continue
+		}
```

Comment:
Unrelated to your PR, but why aren't do doing a map lookup here instead of looping?

Reply:
Good point, I'll fix that.